### ISSUE TITLE
Compatibility with Python >3.6

### ISSUE THREAD

**Issue by Naikless** [SLACKHA/pyJac#41]
Hi,

I noticed that at least the conda package limits compatibility to Python 3.6, explicitly avoiding 3.7.0a0. Has anyone tested whether there is still an issue with more recent python versions? For now, I can just create an environment with 3.6, but I was curious.

**Comment by skyreflectedinmirrors** [comment]
I don't believe there should be any issues with newer python versions, it's just that 3.6 was the most up to date when we were writing the code.

Best,
Nick
________________________________
From: daharn ***@***.***>
Sent: Tuesday, May 11, 2021 8:58 AM
To: SLACKHA/pyJac ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [SLACKHA/pyJac] Compatibility with Python >3.6 (#41)

[CAUTION: External Email]

Hi,

I noticed that at least the conda package limits compatibility to Python 3.6, explicitly avoiding 3.7.0a0. Has anyone tested whether there is still an issue with more recent python versions? For now, I can just create an environment with 3.6, but I was curious.

**Comment by Naikless** [comment]
Wow, thanks for the rapid response!

I will test this as soon as I got it running with my 3.6 environment.

At the moment, I am still having some issues compiling the python interface. I saw that there were issues regarding Windows and the subprocess module. Are those still persisting? I recieve a "File not found" error during execution of subprocess.py.

Edit: So, I pretty much went down the same path as you in https://github.com/SLACKHA/pyJac/issues/32. First had to solve some path issues, but eventually would require some vcvarsall.bat which made be believe I will eventually hit the same incompatibilities as you did. So I switched to Linux.
